app/views/shared/_header.html.haml

```diff
@@ -9,14 +9,15 @@
     .form-group
       = text_field_tag :search,"", placeholder: "Search for videos here", class: "form-control"
       = submit_tag "Search", class: "btn btn-default"
-  #user_links.pull-right
-    %ul
-      %li.dropdown
-        %a(href="#" id="dlabel" role="button" data-toggle="dropdown" class="dropdown-toggle" data-target="#")
-          Welcome, Philip J. Fry
-          %b.caret
-        %ul.dropdown-menu(role="menu" aria-labelledby="dlabel")
-          %li
-            %a(href="#") Account
-            %a(href="#") Plan and Billing
-            %a(href="#") Sign Out
+  = if logged_in?
+    #user_links.pull-right
+      %ul
+        %li.dropdown
+          %a(href="#" id="dlabel" role="button" data-toggle="dropdown" class="dropdown-toggle" data-target="#")
+            = "Welcome, #{ current_user.full_name }"
```

Comment:
`Welcome, #{current_user.full_name}` I think this just works the same - but less quotes.